sources/programs/even-odd/php/how-to-run-the-solution.md

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
This is supposed to describe how to run the existing code sample. It should also indicate how to download and install the language or how to run it in an online compile. Here is an example of the [Even Odd in C article](https://sampleprograms.io/projects/even-odd/c/).